### ISSUE TITLE
admin password suddenly incorrect

### ISSUE THREAD

**Issue by pepitoria** [sismics/docs#609]
Hi,

I have been using teedy through docker in my unraid box for months and I freaking love it. It works wonders. But today my credentials don't work anymore. I am sure my credentials are ok because I use a password manager.

I was using :latest for the image but tried v1.10 as well cannot go through the login. 

The easy thing would be to reset the admin password, how can I achieve this? I am sorry if this is not the right forum to post this and if it isn't, please point me where I can do so.

**Comment by jendib** [comment]
Do you have anything in logs?

**Comment by pepitoria** [comment]
here it is:
[teedy.log](https://github.com/sismics/docs/files/7907156/teedy.log)

**Comment by jendib** [comment]
It seems to be the same problem as #547

Maybe you didn't shutdown the app properly or something like this, but H2 seems to be very sensible to that.

Your best bet is to migrate your database to PostgreSQL as instructed in the last comment of #547.

**Comment by pepitoria** [comment]
Thank you! for some reason the issue was preventing me from logging in with my user. But I found out that I had an active session on another browser.

I created a second user, added to the same group than my previous user and now I can log in and see all the documents. I am only using teedy for myself and this solution solved my issue. 

I am just leaving this here as a possible walkaround if someone finds themselves in the same situation.
